backend/web/workflows/auth.py

```
@@ -0,0 +1,273 @@
+            environment=environment,
+            message=f"Successfully authenticated with {provider_name}"
+            + (f" ({environment})" if environment else ""),
+        )
+
+    except ValueError as e:
+        # Missing required credential fields
+        raise HTTPException(
+            status_code=400,
+            detail=str(e),
+        )
+    except NotImplementedError as e:
```

Ruff (0.14.10)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)